src/ParkChecklist/ParkChecklist.js

```diff
@@ -113,20 +155,115 @@ const ParkChecklist = ({ apiLink }) => {
               <button onClick={handleGoClick}>Go!</button>
             </div>
           </div>
+          <div className="sorting-options">
+            <div className="sort-by-state">
+              <label>
+                <div
+                  className={`dropdown ${dropdownStateOpen ? "open" : ""}`}
+                  onClick={toggleDropdownState}
+                >
+                  <div className="dropdown-toggle-state">Sort by State:</div>
+                  <div className="dropdown-menu-state">
+                    {states.map((stateCode) => (
+                      <label key={stateCode} className="dropdown-item">
+                        <input
+                          type="checkbox"
+                          defaultChecked={selectedStateParks.some(
+                            (park) => park.attributes.states === stateCode
+                          )}
+                          onChange={() => handleStateCheckboxChange(stateCode)}
+                        />
+                        {stateCode}
+                      </label>
+                    ))}
+                  </div>
+                </div>
+              </label>
+            </div>
+            <div className="sort-by-designation">
+              <label>
+                <div
+                  className={`dropdown ${dropdownDesignationOpen ? "open" : ""}`}
+                  onClick={toggleDropdownDesignation}
```

Comment:
holy god thats a lot of code to have to write for sorting, good on you for getting it pumped out!